llvm/test/tools/llvm-objdump/XCOFF/private-headers.test

```diff
@@ -22,29 +19,34 @@ Sections:
     Flags:           [ STYP_DATA ]
     SectionData:     "0000000000000FC0"
 
-# CHECK32:---File Header:
+#      CHECK32:---File Header:
 # CHECK32-NEXT:Magic:              0x1df
 # CHECK32-NEXT:NumberOfSections:   2
-# CHECK32-NEXT:TimeStamp:          None (0)
+# CHECK32-NEXT:Timestamp:          1970-01-01 00:20:34 (1234)
 # CHECK32-NEXT:SymbolTableOffset:  0x0
 # CHECK32-NEXT:SymbolTableEntries: 0
 # CHECK32-NEXT:OptionalHeaderSize: 0x0
 # CHECK32-NEXT:Flags:              0x0
 
-# CHECK64:---File Header:
+#      CHECK64:---File Header:
 # CHECK64-NEXT:Magic:              0x1f7
 # CHECK64-NEXT:NumberOfSections:   2
-# CHECK64-NEXT:TimeStamp:          None (0)
+# CHECK64-NEXT:Timestamp:          1970-01-01 00:20:34 (1234)
 # CHECK64-NEXT:SymbolTableOffset:  0x0
 # CHECK64-NEXT:SymbolTableEntries: 0
 # CHECK64-NEXT:OptionalHeaderSize: 0x0
 # CHECK64-NEXT:Flags:              0x0
 
+
+## Test if the creation time of XCOFF is zero and the number of symbols is negative.
```

Comment:
As the value of the creation time is significant to this test, I'd explicitly specify its value in the YAML, even if it's the same as the default.